frontend/src/routes/PublicRoutes.js

```diff
@@ -8,13 +8,14 @@ import { Redirect, Route, Router, Switch, useHistory } from 'react-router-dom';
 import SLUGS from 'resources/slugs';
 import Registration from 'components/login/Registration.js';
 import LoginComponent from 'components/login/LoginComponent';
+import RecoverPasswordComponent from 'components/login/RecoverPassword';
```

Comment:
Can you please also update the filename? 

you can use `git mv <old_filename> <new_filename>` to do so.

Reply:
just make sure to navigate to the right folder first :)

Reply:
Done

Reply:
Perfect, Thanks!